api.ts

```diff
@@ -211,6 +211,8 @@ export interface SyncTableDef<
   entityName?: string;
   /** See {@link DynamicOptions.defaultAddDynamicColumns} */
   defaultAddDynamicColumns?: boolean;
+  /** See {@link SyncTableOptions.maxUpdateBatchSize} */
```

Comment:
maybe add `/** @hidden */` to these new fields.

Reply:
Oh, cool.  Will add that to all of the definitions.